packages/framework/src/constants/http-headers.constants.ts

```diff
@@ -1,15 +1,15 @@
 export enum HttpHeaderKeysEnum {
-  SIGNATURE = 'x-novu-signature',
-  ANONYMOUS = 'x-novu-anonymous',
-  EXECUTION_DURATION = 'x-novu-execution-duration',
-  SDK_VERSION = 'x-novu-sdk',
-  FRAMEWORK = 'x-novu-framework',
+  SIGNATURE = 'novu-signature',
+  ANONYMOUS = 'novu-anonymous',
+  SIGNATURE_DEPRECATED = 'x-novu-signature',
+  ANONYMOUS_DEPRECATED = 'x-novu-anonymous',
+  NOVU_FRAMEWORK_SDK = 'novu-framework-sdk',
+  NOVU_FRAMEWORK_API = 'novu-framework-api',
+  FRAMEWORK_VERSION = 'novu-framework-version',
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** Suggest to rename all the enum keys (CONSTANT_CASE) to align with the enum values (kebab-case) for maximum explicitness and readability.‏

Reply:
I like this pattern and will apply it. I will also create a task to follow up and move the helper on a shared location so that it can be reused in more locations